helm/templates/ingress.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "immudb.fullname" . -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+
+
+apiVersion: {{ include "immudb.chart.ingressapiversion" . }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}-http
+  labels:
+    {{- include "immudb.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
+  ingressClassName: {{ .Values.ingress.className }}
+  {{- end }}
+  {{- if .Values.ingress.tls }}
+  tls:
+    {{- range .Values.ingress.tls }}
+    - hosts:
+        {{- range .hosts }}
+        - {{ . | quote }}
+        {{- end }}
+      secretName: {{ .secretName }}
+    {{- end }}
+  {{- end }}
+  rules:
+    - host: {{ $.Values.ingress.hostname | quote }}
```

Comment:
It's only one hostname here but the tls section allows specifying multiple hosts - is there any particular reason for that?
Also I'm wondering if hosts in the TLS section couldn't be taken from this value - to make sure those two sections are in sync (at least that's what I've been doing so far in my charts).

Reply:
I've started from a template which allowed multiple hostname in the http section, so that's a leftover from that template. I'll fix that, it has more sense

Reply:
fixed in commit e4c856577131fadfea45ea1cf822acb66c64ccfe